src/alephbet.coffee

```diff
@@ -1,9 +1,10 @@
-utils = require('./utils.js.coffee')
-Storage = require('./storage.js.coffee')
-adapters = require('./adapters.js.coffee')
+utils = require('./utils')
+adapters = require('./adapters')
+options = require('./options')
+LocalStorageAdapter = require('./local_storage_adapter')
```

Comment:
I think this can go into adapters, or we can split adapters into tracking_adapters and storage_adapters and move it there. Otherwise it's inconsistent. I would suggest moving it into adapters for now, and we can refactor later (I would also move the Google adapter there)